README.md

```diff
@@ -1 +1,162 @@
-# Hello-Microverse-Project
\ No newline at end of file
+
+<a name="readme-top"></a>
+
+
+<div align="center">
+  
+  <h3><b>Hello Microverse Project </b></h3>
+
+</div>
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+
+
+# 📖 [Hello Microverse Project] <a name="about-project"></a>
+
+**[Hello Microverse Project]** is the first project we build at Microverse. It just a setup of everything usefule for a web development.
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.w3schools.com/html">HTML</a></li>
+    <li><a href="https://www.w3schools.com/css">CSS</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+
+- **[Setting up Linters properly]**
+- **[Html and css]**
+- **[Mobile view]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+<!-- - [Live Demo Link](https://google.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p> -->
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up, follow these steps:
+- Open your Terminal
+- Navigate to the desired folder
+-git clone https://github.com/rbd3/Hello-Microverse-Project.git
+
+### Prerequisites
+
+In order to run this project you need:
+
+you need a browser like google chrome
+
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+git clone https://github.com/rbd3/Hello-Microverse-Project.git
+```
+### Install
+
+This project doesn't need installation.
```

Comment:
Please add the necessary IDE, tools and other things that you installed during the project